.github/workflows/cd-release.yml

```diff
@@ -0,0 +1,100 @@
+name: CD-release
+
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+
+env:
+  CI: true
+jobs:
+  prepare-deployment:
+    runs-on: ubuntu-20.04
+    outputs:
+      deployment-id: ${{ fromJson(steps.create-deployment.outputs.data).id }}
+    steps:
+      - name: Create GitHub Deployment
+        id: create-deployment
+        uses: octokit/request-action@v2.x
+        with:
+          route: POST /repos/:repository/deployments
+          repository: ${{ github.repository }}
+          ref: ${{ github.sha }}
+          environment: review
+          transient_environment: true
+          auto_merge: false
+          mediaType: '{"previews": ["flash", "ant-man"]}'
+          # The deployment runs in parallel with CI, so status checks will never have succeeded yet:
+          required_contexts: "[]"
+        env:
+          GITHUB_TOKEN: "${{ secrets.GH_TOKEN }}"
+
+  publish-npm:
+    runs-on: ubuntu-20.04
+    needs: [prepare-deployment]
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - name: Mark GitHub Deployment as in progress
+        id: start-deployment
+        uses: octokit/request-action@v2.x
+        with:
+          route: POST /repos/:repository/deployments/:deployment/statuses
+          repository: ${{ github.repository }}
+          deployment: ${{ needs.prepare-deployment.outputs.deployment-id }}
+          environment: review
+          description: "Publishing to npm tag `${GITHUB_REF#refs/tags/v}`…"
+          log_url: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
+          state: in_progress
+          mediaType: '{"previews": ["flash", "ant-man"]}'
+        env:
+          GITHUB_TOKEN: "${{ secrets.GH_TOKEN }}"
+      - name: Prepare for publication to npm
+        uses: actions/setup-node@v2.4.0
+        with:
+          node-version: "12.x"
+          registry-url: "https://registry.npmjs.org"
+      - run: npm ci
+      - name: Publish to npm
+        #  If you want a package to be public (e.g. you'd expect
+        #  `npm publish --access public` here), then just add this to the relevant
+        #  `package.json` file:
+        #    {
+        #      "publishConfig": {
+        #        "access": "public"
+        #      }
+        #    }
+        run: |
+          npm run publish -- from-package --yes 
```

Comment:
This package is public, so from the comment above, shouldn't this `npm run publish` command also include `--access public`...?

Reply:
My understanding of the comment is that including the mentioned snippet in the `package.json` prevents from having to append `--access public` to the NPM command.